lib/griddler/postmark/adapter.rb

```diff
@@ -71,7 +71,7 @@ def attachment_files
       end
 
       def create_tempfile(attachment)
-        filename = attachment[:Name].gsub(/\/|\\/, '_')
+        filename = attachment[:Name].gsub(/\/|\\/, '_')[0..150]
```

Comment:
So keeping the extension doesn't matter because it's just for the tempfile, right?